app/views/users/sessions/new.html.erb

```diff
@@ -13,6 +13,15 @@
         <%= f.label :password, "Heslo" %>
         <%= f.password_field :password, autocomplete: "off", class: "login-text" %>
 
+        <% if devise_mapping.rememberable? -%>
+            <%#= f.check_box :remember_me %>
```

Comment:
Takyto zakomentovany kod prec (ked je pod tym to iste inak, toto uz nebudeme potrebovat odkomentovat).